src/table/utils/handle-click.ts

```diff
@@ -39,30 +39,29 @@ export const getSelectionMouseHandlers = (
   cell: Cell,
   announce: Announce,
   onMouseDown: React.MouseEventHandler<HTMLTableCellElement> | undefined,
-  selectionDispatch: SelectionDispatch,
-  areBasicFeaturesEnabled: boolean
+  selectionDispatch: SelectionDispatch
 ) => {
   const handleMouseDown = (evt: React.MouseEvent) => {
     // run handleClickToFocusBody
     onMouseDown?.(evt as React.MouseEvent<HTMLTableCellElement>);
     // only need to check isSelectable here. once you are holding you want to be able to drag outside the current column
-    if (areBasicFeaturesEnabled && cell.isSelectable) {
+    if (cell.isSelectable) {
       const mouseupOutsideCallback = () => selectionDispatch({ type: SelectionActions.SELECT_MULTI_END });
       selectionDispatch({ type: SelectionActions.SELECT_MOUSE_DOWN, payload: { cell, mouseupOutsideCallback } });
     }
   };
 
   const handleMouseOver = (evt: React.MouseEvent) => {
-    if (areBasicFeaturesEnabled && evt.buttons === 1)
+    if (evt.buttons === 1)
       selectionDispatch({ type: SelectionActions.SELECT_MULTI_ADD, payload: { cell, evt, announce } });
   };
 
   const handleMouseUp = (evt: React.MouseEvent) => {
     if (evt.button === 0) {
-      if (areBasicFeaturesEnabled) {
-        selectionDispatch({ type: SelectionActions.SELECT_MOUSE_UP, payload: { cell, evt, announce } });
-      } else if (cell.isSelectable) {
+      if (cell.isSelectable) {
         selectionDispatch({ type: SelectionActions.SELECT, payload: { cell, evt, announce } });
+      } else {
+        selectionDispatch({ type: SelectionActions.SELECT_MOUSE_UP, payload: { cell, evt, announce } });
       }
     }
   };
```

Comment:
Is this should be like my change? 
Also it's tests I'm a bit confused if it should be like my changes or not. 

Pls pay a bit close attention to this part and let me know if something is wrong 🙏

Reply:
you should think of it as if `areBasicFeaturesEnabled` is always true. That means that the else block will never run and you can remove it

Reply:
Then the entire`SelectionActions.SELECT_MOUSE_UP` becomes obsolete? (i mean in `selection-utils.js`)

Reply:
I meant the opposite. If you look at the original code and areBasicFeaturesEnabled is always true. You can remove the else block, meaning `SelectionActions.SELECT` is never called from here.

Reply:
this is what you get left
```
if (evt.button === 0) {
  selectionDispatch({ type: SelectionActions.SELECT_MOUSE_UP, payload: { cell, evt, announce } });
}